sys-apps/turnstile/turnstile-0.1.10.ebuild

```diff
@@ -20,6 +20,7 @@ LICENSE="BSD-2"
 SLOT="0"
 IUSE="dinit runit rundir man"
 
+BDEPEND="app-text/scdoc"
```

Comment:
This doesn't appear to be necessary for `USE="-man"` - am I missing something? If not, please gatekeep this behind `USE="man"`.